docs/Makefile

```diff
@@ -23,6 +24,14 @@ help:
 clean:
 	rm -rf $(BUILDDIR)/* $(AUTODOCDIR)
 
+markdown:
+	pip install -r ../requirements.txt
+	pip install -r ../requirements.dev.txt
+	pip install -r ./requirements.txt
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+	./make_markdown.sh
```

Comment:
Moved into it's on bash script since it became pretty complex and I wanted to add comments